python/gui/qgsstatusbarmagnifierwidget.sip

```diff
@@ -0,0 +1,32 @@
+class QgsStatusBarMagnifierWidget: QWidget
+{
+%TypeHeaderCode
+#include <qgsstatusbarmagnifierwidget.h>
+%End
+  public:
+
+    /** Constructor
+      * @param parent is the parent widget
+      * @param canvas the map canvas
+      */
+    QgsStatusBarMagnifierWidget( QWidget* parent, QgsMapCanvas *canvas );
+
+    /** Destructor */
+    virtual ~QgsStatusBarMagnifierWidget();
+
+    /** Returns the current magnification level
+      * @return magnification level
+      */
+    double magnificationLevel();
+
+    /** Set the font of the text
+      * @param font the font to use
+      */
+    void setFont( const QFont& font );
+
+    /** Set the magnification level
+      * @param level the magnification level
+      * @return true if the level is valid, false otherwise
+      */
+    bool setMagnificationLevel( const double level );
```

Comment:
Actually, you're right, it's not a necessity. But, IMHO, it's a good practice to indicate that a value is not changed within a method.